x/liquidvesting/keeper/msg_server_test.go

```diff
@@ -231,22 +231,20 @@ func (suite *KeeperTestSuite) TestLiquidate() {
 			suite.T().Logf("spendable coins: %s", spendable.String())
 			suite.T().Logf("liquidation amount: %s", tc.amount.String())
 
-			resp, err := suite.network.App.LiquidVestingKeeper.Liquidate(ctx, fromAddr, toAddr, tc.amount)
-			expResponse := &types.MsgLiquidateResponse{
-				Minted: sdk.NewCoin(types.DenomBaseNameFromID(0), tc.amount.Amount),
-			}
+			burnt, contractAddr, err := suite.network.App.LiquidVestingKeeper.Liquidate(ctx, fromAddr, toAddr, tc.amount)
```

Comment:
This method doesn't burn coins. It returns `minted` `aLIQUIDx` coins. Corresponding `aISLM` coins are being transferred to liquid vesting module account.